libs/components/theme/src/lib/theming/theme.service.ts

```diff
@@ -206,7 +206,7 @@ export class SkyThemeService {
       default:
     }
 
-    if (supportedValues && !supportedValues.includes(value)) {
+    if (supportedValues && value && !supportedValues.includes(value as any)) {
```

Comment:
Casting as any is bad practice. Change this code so it doesn't require to cast as any. You may also need to run the lint step to ensure you are not introducing any other linting errors.